### ISSUE TITLE
Error updating a meeting

### ISSUE THREAD

**Issue by jkudish** [MacsiDigital/laravel-zoom#52]
Hi @colinhall17 

Thanks for this great package. I have encountered a small bug while attempting to update meetings. I have a proposed fix for this as well, outlined below. Thanks!

- API Version: not sure which API version you mean? Running `macsidigital/laravel-zoom` v.4.0.9 and `macsidigital/laravel-api-client` v3.1.2 which seem to be the latest versions of each
- Laravel Version: v7.22.4
- PHP Version: 7.4.6

**Describe the bug**

When updating a Zoom meeting, I get the following error:

`Error : Call to undefined method MacsiDigital\Zoom\TrackingField::setPassOnAttributes()`

With the following backtrace (I think phpunit cuts off the full backtrace in my setup)

```
.../vendor/macsidigital/laravel-api-client/src/Support/Relations/HasMany.php:163
.../vendor/macsidigital/laravel-api-client/src/Support/Relations/HasMany.php:146
.../vendor/macsidigital/laravel-api-client/src/Support/PersistResource.php:81
.../vendor/macsidigital/laravel-api-client/src/Support/PersistResource.php:24
.../vendor/macsidigital/laravel-api-client/src/Traits/InteractsWithAPI.php:389
.../vendor/macsidigital/laravel-api-client/src/Traits/InteractsWithAPI.php:341
.../vendor/macsidigital/laravel-api-client/src/Traits/InteractsWithAPI.php:273
```

**To Reproduce**
Here's a barebones test that reproduces the issue:

```php
/** @test */
public function it_can_update_a_meeting()
{
    Carbon::setTestNow('August 3 5:00 pm');
    $zoom = app('zoom');

    $user = $zoom->user->find('info@imagesalon.com');

    $meeting = $user->meetings()->create([
        'type' => 2,
        'topic' => 'Testing',
        'agenda' => 'Testing',
        'start_time' => now()->toDateTimeLocalString(),
        'timezone' => 'America/Montreal',
        'duration' => 30,
        'password' => Str::random(10),
    ]);

    $meeting->update([
        'topic' => 'Testing --updated',
        'agenda' => 'Testing --updated',
        'start_time' => now()->addMinutes(30)->toDateTimeLocalString(),
        'timezone' => 'America/Montreal',
        'duration' => 45,
    ]);
}
```

**Expected behavior**
The meeting should update. But it doesn't, due to the fatal error that occurs.

**Additional context**
I believe the fix is to change `src/TrackingField.php` to extend `MacsiDigital\API\Support\ApiResource` instead of `MacsiDigital\API\Support\Resource`. When I did that, I was able to fix my code. I am hoping validate the idea with you as I'm not as familiar with the package's code. I submitted a PR in #53 for you to test it out and review

**Comment by colinhall17** [comment]
Hi @jkudish 

Thanks for reaching out.

We have applied a fix to the underlying API library.  The ApiResource is any resource that can communicate directly through the API, as the tracking field cant we needed to find the underlying issue on our API library.

We have now done this and patched so it checks before trying to call this method.

If you update composer it should pull in the new version of the API Library.

Thanks again
